stories/constants/colorsStory.js

```diff
@@ -22,7 +21,7 @@ const ColorsContainer = styled.div`
 const colorsStory = () => (
   <MainContainer>
     {Object.keys(COLORS).map(category => {
-      if (category === 'default') {
+      if (category === 'default' || category === 'COLORS_PROP_TYPES') {
```

Comment:
Pushed this fix. Test in real app works good, we do have the new export `COLORS_PROP_TYPES` in the constants (intellisense, etc) now but it's ok

Reply:
@daigof Constants intellisense for `PocketDerm` or this repo?  If PocketDerm we can remove the prop export from the index file, and just import it directly. Would that help?

Reply:
as of now it is an exported member so we have it available in PocketDerm. `remove the prop export ` would help but I didnt consider it a big deal, note that if you opt to remove the export then we may not need my fix anymore